### DISCUSSION TITLE
アカウント登録の仕方について　How to register an account

### DISCUSSION THREAD

**Discussion by takuyatahata** [chocolatey/home#255] [Q&A]
Is there a way to log in from an account I already have, such as Gmail?
Ｇmailなどの既に持っているアカウントからログイン出来る方法はありますか？

**Comment by TheCakeIsNaOH** [comment]
Are you talking about a community.chocolatey.org account? If so, then no, there are not currently any single sign on options like google.

Go to https://community.chocolatey.org/account/Register to register.
